### DISCUSSION TITLE
Exists a utility class analogous to the Tailwind 'container' class?

### DISCUSSION THREAD

**Discussion by Andres6936** [master-co/css#281] [Question]
In Tailwind CSS this class exists: https://tailwindcss.com/docs/container, Exists a utility or analogous class in Master CSS?

**Comment by 1aron** [comment]
@Andres6936 It is equivalent to https://beta.css.master.co/docs/max-width:
```html
<div class="max-w:md@lg max-w:lg@xl ... mx:auto">
```
